apps/account-api/src/controllers/v1/delegation-v1.controller.ts

```diff
@@ -28,4 +50,51 @@ export class DelegationControllerV1 {
       throw new HttpException('Failed to find the delegation', HttpStatus.BAD_REQUEST);
     }
   }
+
+  @Get('revokeDelegation/:accountId/:providerId')
+  @HttpCode(HttpStatus.OK)
+  @ApiOperation({ summary: 'Get a properly encoded RevokeDelegationPayload that can be signed' })
+  @ApiOkResponse({ description: 'Returned an encoded RevokeDelegationPayload for signing' })
```

Comment:
add `type: RevokeDelegationPayloadResponseDto` to `ApiOkResponse()` so that the response payload is included in the generated OpenAPI spec.